hw03_frequency_analysis/go.mod

```
@@ -1,10 +1,9 @@
-module github.com/fixme_my_friend/hw03_frequency_analysis
+module github.com/IMLogicI/-hw_otus/tree/master/hw03_frequency_analysis
```

```suggestion
module github.com/IMLogicI/-hw_otus/hw03_frequency_analysis
```
`/tree/master` - это роутинг UI GitHub, фактически не имеет отношения к модулям Go

---

Поправлено